backend/xivdpscalc/pps/sch.py

```diff
@@ -104,6 +100,10 @@ def get_total_potency_variable_time(self, sim_length: ElapsedTime, character_sta
             elif selected_action == SchAction.BIOLYSIS:
                 active_effects[SchEffect.BIOLYSIS] = current_time + 30
 
+            # advance time based on cast time or animation lock as needed
+            current_time += \
+                max(cast_time + caster_tax, animation_lock)
```

Comment:
nit: does this really need a newline to prevent linter freakout?

Reply:
Yeah, unless the linter is complaining, I don't think this line should be split.